[2024-11-18T12:55:56.153+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-18T12:55:56.236+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: raw_api_sales_data.test manual__2024-11-18T12:55:43.208505+00:00 [queued]>
[2024-11-18T12:55:56.279+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: raw_api_sales_data.test manual__2024-11-18T12:55:43.208505+00:00 [queued]>
[2024-11-18T12:55:56.280+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2024-11-18T12:55:56.341+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): test> on 2024-11-18 12:55:43.208505+00:00
[2024-11-18T12:55:56.358+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=303) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-11-18T12:55:56.363+0000] {standard_task_runner.py:72} INFO - Started process 305 to run task
[2024-11-18T12:55:56.365+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'raw_api_sales_data', 'test', 'manual__2024-11-18T12:55:43.208505+00:00', '--job-id', '336', '--raw', '--subdir', 'DAGS_FOLDER/fetch_data.py', '--cfg-path', '/tmp/tmp2ms41pn1']
[2024-11-18T12:55:56.369+0000] {standard_task_runner.py:105} INFO - Job 336: Subtask test
[2024-11-18T12:55:56.515+0000] {task_command.py:467} INFO - Running <TaskInstance: raw_api_sales_data.test manual__2024-11-18T12:55:43.208505+00:00 [running]> on host c06adc74012b
[2024-11-18T12:55:57.341+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='raw_api_sales_data' AIRFLOW_CTX_TASK_ID='test' AIRFLOW_CTX_EXECUTION_DATE='2024-11-18T12:55:43.208505+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-18T12:55:43.208505+00:00'
[2024-11-18T12:55:57.345+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-11-18T12:55:57.347+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-11-18T12:55:57.349+0000] {logging_mixin.py:190} INFO - Current task name:test state:running start_date:2024-11-18 12:55:56.240309+00:00
[2024-11-18T12:55:57.351+0000] {logging_mixin.py:190} INFO - Dag name:raw_api_sales_data and current dag run status:running
[2024-11-18T12:55:57.353+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-18T12:55:57.389+0000] {base.py:84} INFO - Retrieving connection 'postgres_default'
[2024-11-18T12:55:57.441+0000] {logging_mixin.py:190} INFO - PostgreSQL connection successful!
[2024-11-18T12:55:57.444+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-11-18T12:55:57.504+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-18T12:55:57.507+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=raw_api_sales_data, task_id=test, run_id=manual__2024-11-18T12:55:43.208505+00:00, execution_date=20241118T125543, start_date=20241118T125556, end_date=20241118T125557
[2024-11-18T12:55:57.586+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2024-11-18T12:55:57.588+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2024-11-18T12:55:57.592+0000] {logging_mixin.py:190} INFO - Dag name:raw_api_sales_data queued_at:2024-11-18 12:55:43.251040+00:00
[2024-11-18T12:55:57.594+0000] {logging_mixin.py:190} INFO - Task hostname:c06adc74012b operator:PythonOperator
[2024-11-18T12:55:57.632+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-18T12:55:57.705+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-11-18T12:55:57.723+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
